openedx/features/philu_courseware/views.py

```diff
@@ -1,10 +1,9 @@
 """
 Views to add features in courseware.
 """
-
 from django.utils.translation import ugettext as _
 from opaque_keys import InvalidKeyError
-from opaque_keys.edx.django.models import CourseKey, UsageKey
+from opaque_keys.edx.django.models import CourseKey
 from rest_framework import status
 from rest_framework.exceptions import APIException, ValidationError
 from rest_framework.response import Response
```

Comment:
Shouldn't we separate opaque key imports from django and DRF imports?

Reply:
We run isort and do not manage imports ordering and spacing manually. This is the result of isort.